scripts/export_environment.py

```diff
@@ -264,19 +264,42 @@ def resolve_env(env: Environment) -> Environment:
     return dict(ResolvedEnvironment(env))
 
 
+azul_env_vars = 'AZUL_ENV_VARS'
```

Comment:
I don't think we want to export this to the Lambda function environment, so it should be lower-case.